### PR TITLE
Prerequisites added in the docs

### PR DESCRIPTION
**Pull request by ansrivas**
Added prerequisites steps for creating dapps. Minor changes in gettingStarted markdown file.

## Description

Getting started document didn't mention about the nOS browser building part. Added a few lines of explanation for that.
## Motivation and Context

Just trying to add more clarity to the docs, as I couldn't figure out how can nOS browser be used to run the apps.

## Types of changes

- [ ] Chore (tests, refactors, and fixes)
- [ ] New feature (adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] I have read the **CONTRIBUTING** document.

## Closing issues

Fixes #

### CODE REVIEW

File: docs/create-your-own-dapp.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # Creating Your First nOS dApp
 
+### Pre-requisites:
+
+First of all you'll need to build nOS browser locally, follow these instructions here:
+
+```
+// Install dependencies and launch the nOS client
```
Comment (jeroenptrs) [inline docs/create-your-own-dapp.md]: Might want to add a `git clone` as well.

File: docs/create-your-own-dapp.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
 # Creating Your First nOS dApp
 
+### Pre-requisites:
+
+First of all you'll need to build nOS browser locally, follow these instructions here:
```
Comment (jeroenptrs) [inline docs/create-your-own-dapp.md]: Good additions would be to **stress** that you need the nOS client to open your dApp and where you can get it.

File: docs/create-your-own-dapp.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+// This is used to start developing on the nOS client
+yarn install && yarn start
+
+// or if you want to create a temporary binary
+
+// Distribution command
+// After running `yarn dist` you can navigate to `dist` directory and execute the `nOS` binary in a terminal.
+yarn install && yarn dist
+```
+
```
Comment (jeroenptrs) [inline docs/create-your-own-dapp.md]: The @nosplatform/api-functions NPM package might be a good addition to the prerequisites.

Comment (ansrivas) [reply docs/create-your-own-dapp.md]: Hi @jeroenptrs thank you for the comments. 
Didn't know that NPM package existed for nos. Can you point me to it?

Comment (DalderupMaurice) [reply docs/create-your-own-dapp.md]: The nOS api-functions are meant for the dApp within nOS, not nOS itself :D

Comment (jeroenptrs) [reply docs/create-your-own-dapp.md]: @DalderupMaurice correct. @ansrivas sorry for not being clear, the NPM package is used in your dApp, not the client. It's baked into the starter kit, but if you want to make a dApp from scratch (without the starter kit templates) you'll probably need the package.

**Comment by ansrivas** [comment]
@DalderupMaurice I was thinking exactly the same, but the dApp starter kit already has it in `package.json`   [here](https://github.com/nos/dapp-starter-kit/blob/c161ef4c4e6454bcf00619ec5f0fd91b398f1e52/package.json#L36) right. That's where I got confused and added rather to the instructions. Should I remove it rather?

**Comment by jeroenptrs** [comment]
It's baked into the starter kit, but if you want to make a dApp from scratch (without the starter kit templates) you'll probably need the package. This should be clear in this document @ansrivas

**Pull request merged**